### ISSUE TITLE
Some basic tips

### ISSUE THREAD

**Issue by karolstawowski** [chipmunk-dev/to-do-list#2]
Hello!

I've been searching for `good first issue` labels on github and found your repository. I have to admit, you look like a good material to become a great programmer. I would like to share some tips with you:

- Personally I prefer to use [Yarn](https://yarnpkg.com/) instead of npm as a packages manager - it installs dependency packages in parallel, so it means faster install time in case of big projects. It would be better if you learn useful things at the beginning of your dev road, trust me.
- I have noticed you're using environmental variables - it would be nice for the person, which uses your repository, to know what variables the `.env` file should contain. To help a user, you can create `.env.example` file with names of variables from `.env` file, but without any values assigned to them, eg.

`.env.example`:
```bash
DB_USER=
DB_PASSWORD=
BACKEND_URL=
```

Hope it will help you a little bit in the future!

Have a nice weekend!

Greetings,
Karol

**Comment by chipmunk-dev** [comment]
@karolstawowski 
wow.. thank you so much
I feel like I've been given a surprise present. Your advice was just what I needed.
I will reflect the two advices you gave.

thank you :)

**Comment by chipmunk-dev** [comment]
apply advice `create .env.example` in #5 
However, using 'yarn' is likely to be difficult in this project. 
In the new project, we will manage the package using 'yarn'.

**Comment by karolstawowski** [comment]
I can see you added `.env.example` file - nice job!

Regarding `yarn` - you can add `yarn` to your project at the moment by:

1. deleting `package-lock.json` file
2. changing dependencies versions in `package.json` to match the exact version (delete `^` in versions) - it is related to [semantic versioning](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies) (you will learn about it later on, but you just want to avoid using newer packages than yours)
3. deleting `node_modules` directory
4. using `yarn` command to install packages

Then you just use `yarn` commands, eg.

- `npm install`-> `yarn`
- `npm uninstall [package]` -> `yarn remove [package]`
- `npm init` == `yarn init`

[Here are `npm` and `yarn` differences](https://www.digitalocean.com/community/tutorials/nodejs-npm-yarn-cheatsheet)

Of course it is up to you to decide if you're going to use yarn in this project 😄 

If you have any questions - feel free to write to me.

Lastly - where do you learn these stuff from? I'm just curious, because the quality of the code is pretty nice and I like it 😀

안녕~

**Comment by chipmunk-dev** [comment]
I have reviewed your response.
Thank you for providing detailed information on how to switch from npm to yarn.
I have created an issue for this matter #8 .
I have reopened the issue you have provided #8  and will close it when resolving (issue number).
Thank you for reviewing the code and providing feedback :)

Additionally, the code style I used was learned from an online course created by a leading developer working in Australia. Here is the website for the course:
https://academy.dream-coding.com/courses/node

Also, I am learning by searching on Google and reading blog posts when I have questions or don't know something.
